pkg/crosscluster/logical/testdata/ldr_statements

```diff
@@ -0,0 +1,153 @@
+exec
+CREATE TABLE basic_table (
+  id INT PRIMARY KEY,
+  name STRING,
+  value INT,
+  data BYTES
+);
+----
+ok
+
+show-insert table=basic_table
+----
+INSERT INTO [104 AS replication_target](id, name, value, data) VALUES ($1, $2, $3, $4)
+
+show-update table=basic_table
+----
+UPDATE [104 AS replication_target] SET id = $5::INT8, name = $6::STRING, value = $7::INT8, data = $8::BYTES WHERE (((id = $1) AND (name = $2)) AND (value = $3)) AND (data = $4)
+
+show-delete table=basic_table
+----
+DELETE FROM [104 AS replication_target] WHERE (((id = $1) AND (name = $2)) AND (value = $3)) AND (data = $4)
+
+exec
+CREATE TYPE status AS ENUM ('pending', 'active', 'completed', 'cancelled');
+----
+ok
+
+# Create a table that uses a user defined type.
```

Comment:
Nice!

That reminds me, does LDR prohibit ALTER TYPE statements on both clusters?

Reply:
This is a bug we discovered last week 😅  We need to backport a change that prevents users from modifying types referenced by a replicated table.